source/fleetprovisioning/FleetProvisioning.h

```diff
@@ -115,10 +115,17 @@ namespace Aws
                      * \brief the location where keys generated by Fleet Provisioning will be stored
                      */
                     std::string keyDir = Config::DEFAULT_KEY_DIR;
+
+                    /**
+                     * \brief stores device config of newly provisioned device
```

Comment:
What is a device config?

Reply:
The device configuration contains arbitrary data user want to send to your devices when provisioning. More details are available [here](https://docs.aws.amazon.com/iot/latest/developerguide/provision-template.html#device-config).

Reply:
Maybe include some of that info in this doxygen comment? Otherwise the reader cannot get any valuable info from this comment

Reply:
Done.